runtime/mainnet/src/lib.rs

```diff
@@ -77,6 +72,12 @@ pub type AccountId = <<Signature as Verify>::Signer as IdentifyAccount>::Account
 /// The address format for describing accounts.
 pub type Address = MultiAddress<AccountId, ()>;
 
+/// Record of an event happening.
+pub type EventRecord = frame_system::EventRecord<
```

Comment:
It seems this type is not being used.

Reply:
https://github.com/r0gue-io/pop-node/pull/447/commits/c23cf626382dd98b66e27fbe5c921099fb6ae1c5